test/uk/gov/hmrc/merchandiseinbaggage/stubs/MibBackendStub.scala

```diff
@@ -59,15 +60,15 @@ object MibBackendStub extends MibConfiguration with CoreTestData {
     val declarationWithId = declaration.copy(declarationId = declarationId)
     server
       .stubFor(
-        get(urlPathEqualTo(s"$declarationsUrl/${declarationWithId.declarationId.value}"))
+        get(urlPathEqualTo(s"${config.mibDeclarationsUrl}/${declarationWithId.declarationId.value}"))
           .willReturn(okJson(Json.toJson(declarationWithId).toString))
       )
   }
 
   def givenPersistedDeclarationIsFound()(implicit server: WireMockServer): StubMapping =
     server
       .stubFor(
-        get(urlMatching(s"$declarationsUrl/(.*)"))
+        get(urlMatching(s"${config.mibDeclarationsUrl}/(.*)"))
```

Comment:
This can just be `get(urlMatching("/declare-commercial-goods/declarations/(.*)"))`